client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -4,170 +4,294 @@
-import { find } from 'lodash';
+import debugFactory from 'debug';
+import { find, size } from 'lodash';
 import { localize } from 'i18n-calypso';
+const debug = debugFactory( 'calypso:allendav' );
 
 /**
  * Internal dependencies
  */
-import { installPlugin, activatePlugin, fetchPlugins } from 'state/plugins/installed/actions';
+import { installPlugin } from 'state/plugins/installed/actions';
 import { fetchPluginData } from 'state/plugins/wporg/actions';
 import { getPlugin } from 'state/plugins/wporg/selectors';
 import { getPlugins } from 'state/plugins/installed/selectors';
 import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
 import ProgressBar from 'components/progress-bar';
 import QueryJetpackPlugins from 'components/data/query-jetpack-plugins';
 import SetupHeader from './setup-header';
-import { setFinishedInstallOfRequiredPlugins } from 'woocommerce/state/sites/setup-choices/actions';
-import wp from 'lib/wp';
-
-const requiredPlugins = [
-	'wc-api-dev',
-	'woocommerce',
-	'woocommerce-gateway-stripe',
-	'woocommerce-services',
-	'taxjar-simplified-taxes-for-woocommerce',
-];
+//import { setFinishedInstallOfRequiredPlugins } from 'woocommerce/state/sites/setup-choices/actions';
+// import wp from 'lib/wp';
 
 class RequiredPluginsInstallView extends Component {
 	static propTypes = {
-		fetchPluginData: PropTypes.func.isRequired,
-		installPlugin: PropTypes.func.isRequired,
-		plugins: PropTypes.array,
-		setFinishedInstallOfRequiredPlugins: PropTypes.func.isRequired,
 		site: PropTypes.shape( {
-			ID: PropTypes.number.isRequired,
-		} ),
-		wpOrg: PropTypes.array,
+			ID: PropTypes.number.isRequired
+		} )
 	};
 
 	constructor( props ) {
 		super( props );
 		this.state = {
-			activatingPlugin: null,
-			installingPlugin: null,
+			engineState: 'INITIALIZING',
+			message: '',
 			progress: 0,
+			toActivate: [],
+			toInstall: [],
+			workingOn: ''
 		};
+		this.updateTimer = false;
 	}
 
 	componentDidMount = () => {
-		const { plugins, site } = this.props;
+		this.createUpdateTimer();
+	}
+
+	componentWillUnmount = () => {
+		this.destroyUpdateTimer();
+	}
 
-		if ( site && plugins && plugins.length ) {
-			this.installPlugins( plugins );
+	createUpdateTimer = () => {
+		if ( this.updateTimer ) {
+			return;
 		}
 
-		this.getWporgPluginData();
+		this.updateTimer = window.setInterval( () => {
+			this.updateEngine();
+		}, 1000 );
 	}
 
-	componentWillUnmount = () => {
-		this.cancelUpdateTimeout();
+	destroyUpdateTimer = () => {
+		if ( this.updateTimer ) {
+			window.clearInterval( this.updateTimer );
+			this.updateTimer = false;
+		}
+	}
+
+	getRequiredPluginsList = () => {
+		const { translate } = this.props;
+
+		return {
+			woocommerce: translate( 'WooCommerce' ),
+			'wc-api-dev': translate( 'WooCommerce API Dev' ),
+			'woocommerce-gateway-stripe': translate( 'WooCommerce Stripe Gateway' ),
+			'woocommerce-services': translate( 'WooCommerce Services' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Install WooCommerce Services' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/15901-plugin-install-errors)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).